### DISCUSSION TITLE
With this i can show the cover album of some audio file?

### DISCUSSION THREAD

**Discussion by JhonCODEOWO** [mono/taglib-sharp#315] [Q&A]
Somebody can tell me if this API help me to show the cover album from some audio file please?

**Comment by Vasilich** [comment]
if you mean "album cover", then specify file types, from which do you expect to get covers,
Different file types keep covers in different places. E.g. for ID3v2 there is a property "Pictures" as an array of IPicture.
And what is your problem with API?

**Comment by Starwer** [comment]
With TagLib#, you can get a list of all the images embedded in a media file using property `Pictures`. The album cover may be one of these.

**Comment by sharpchen** [comment]
If you want to write album cover to a file, you can try
```Csharp
var data = TagLib.File.Create(path).Tag.Pictures[index].Data.Data;
File.WriteAllBytes(outputPath, data);
```
